src/index.html

```diff
@@ -9,6 +9,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav__item">
+            <a href="#" class="nav_link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="is-active">
```

Comment:
not all links should be active according to the design, on;y the first one, all others have nav__link class

> the link with blue color and line below is an active link. It should have class="is-active" and relevant styles.
```suggestion
            <a href="#" class="nav__link">
```